test/test.js

```diff
@@ -0,0 +1,36 @@
+const tap = require('tap');
+const request = require('request')
+const Fastify = require('fastify');
+const fastifyJitson = require('../fastify-jitson');
+
```

Comment:
Can you add another test using proxyquire, so we validate jitson is actually getting called?